### PR TITLE
Convert role into collection

### PR DESCRIPTION
**Pull request by l00ptr**
We move the role files under the roles folder and we also adapt the
molecule default scenario and the documentation. It's the first step before adding some plugins to manipulate the config through the LemonLDAP::NG API.

### CODE REVIEW

File: galaxy.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+namespace: worteks
+
+# The name of the collection. Has the same character restrictions as 'namespace'
+name: lemonldap
+
+# The version of the collection. Must be compatible with semantic versioning
+version: 0.0.1
+
+# The path to the Markdown (.md) readme file. This path is relative to the root of the collection
+readme: README.md
```
Comment (maxbes) [inline galaxy.yml:16]: Please update the README to reflect the new role path in the example

**Review by maxbes — CHANGES_REQUESTED** [review]

**Comment by l00ptr** [comment]
Any feedback for this (long standing) MR ?

**Review by maxbes — APPROVED** [review]

**Comment by maxbes** [comment]
@coudot looks good to me

**Comment by l00ptr** [comment]
I will have some free time to work on some improvement / maintenance on this collection the 2 next weeks. I will create a new branch starting from this unmerged one.

**Comment by coudot** [comment]
Hello @l00ptr 

following review from @maxbes I merged this PR. You can start a new one from master branch if you want.

Thanks again for your contribution.

**Comment by l00ptr** [comment]
@coudot great thx!

**Pull request merged**